src/ape_compile/_cli.py

```diff
@@ -10,12 +10,34 @@
 _flatten = chain.from_iterable
 
 
+def _create_contracts_paths(ctx, param, value):
+    from ape import project
```

Comment:
We can just use the `ctx` param here if `https://github.com/ApeWorX/ape/pull/246` merges